packages/forms/src/UIForm/Widget/Widget.component.js

```diff
@@ -47,6 +57,25 @@ export default function Widget(props) {
 	const error = getError(props.errors, props.schema);
 	const errorMessage = validationMessage || error;
 
+	if (tooltip) {
+		return (
+			<TooltipTrigger label={tooltip} tooltipPlacement={'left'}>
```

Comment:
I'm not convinced by the position, it fits in TDP but we should let the possibility to change it

Reply:
+1